### ISSUE TITLE
Freeform 5: table field not rendering correctly

### ISSUE THREAD

**Issue by WHITE-developer** [solspace/craft3-freeform#1277]
The settings in, for example `bootstrap-5/fields/table.twig` , aren't taken into account ending up with a unstyled table field in the frontend.
Also, `addButtonLabel` and `removeButtonLabel` shouldn't be overridden by default as there is a setting in the form for that.

**Craft & Plugin Info (please complete the following information):**

- Craft Version:
- Freeform Version: 5.1.19.1
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh

**Additional context**
![Screenshot from 2024-05-15 10-58-50](https://github.com/solspace/craft-freeform/assets/22771556/ad0aeac1-be8f-448a-8232-78855bfda094)

**Comment by kjmartens** [comment]
Sorry about this @WHITE-developer,

I know that those template overrides worked at some point. Not sure when they stopped working, but we'll investigate the issue and get a fix for it shortly. 🙂 

As for forcing overrides for `addButtonLabel` and `removeButtonLabel` on Table fields, you're right about that. We will remove those as well.

**Comment by theurbanu** [comment]
Plus one to this. It's happening for me as well on a fresh install. Hopefully this gets fixed quickly as I need to launch this form on Monday

**Comment by kjmartens** [comment]
@theurbanu we hope to have a fix available by tomorrow. It's also worth noting that if you need a solution right away, it's still possible to control the display via CSS with something like: 🙂 

``` CSS
.freeform-fieldtype-table table {
    width: 100%;
    font: 400 14px sans-serif;
    text-align: left;
}
.freeform-fieldtype-table table thead tr th {
    padding-bottom: 10px;
}
.freeform-fieldtype-table table tbody tr:only-child button {
    display: none;
}
.freeform-fieldtype-table table tbody tr td {
    padding: 0 10px 10px 0;
}
.freeform-fieldtype-table table tbody tr td:last-child {
    text-align: right;
    padding-right: 0;
}
.freeform-fieldtype-table table button[data-freeform-table-remove-row] {
    font-size: 1px;
    margin: 0;
    padding: 0;
    visibility: hidden;
}
.freeform-fieldtype-table table button[data-freeform-table-remove-row]:after {
    font: 400 14px sans-serif;
    color: #b02a37;
    line-height: 1.5;
    content: "✕";
    border: 1px solid #b02a37;
    border-radius: 5px;
    padding: 5px 15px;
    display: inline-block;
    vertical-align: middle;
    cursor: pointer;
    visibility: visible;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out;
}
.freeform-fieldtype-table table button[data-freeform-table-remove-row]:hover:after {
    color: #ffffff;
    background-color: #b02a37;
    border-color: #b02a37;
}
.freeform-fieldtype-table button[data-freeform-table-add-row] {
    font: 400 14px sans-serif;
    line-height: 1.5;
    color: #146c43;
    content: "+";
    border: 1px solid #146c43;
    background-color: transparent;
    margin: 0;
}
.freeform-fieldtype-table button[data-freeform-table-add-row]:hover {
    color: #ffffff;
    background-color: #146c43;
    border-color: #146c43;
}
.freeform-fieldtype-table button[data-freeform-table-add-row]:after {
    content: "+";
    margin-left: 5px;
}
.freeform-fieldtype-table input[type="checkbox"] {
    display: block;
    transform: scale(1.15);
}
.freeform-fieldtype-table select {
    min-width: 200px;
}
```

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.3.0+ 🙂
